### DISCUSSION TITLE
OpenAI API Key not being used.

### DISCUSSION THREAD

**Discussion by sigott** [RamiAwar/dataline#321] [Q&A] [answered]
I don't know if this is the best place to post a bug report but I couldn't create an issue so I've come here.  
I setup a Mysql connection with 2 tables and tried to ask some very simple questions but I consistently get a "User Quota Exceeded Error"  Langsmith seems to work and if I view those logs, the error is coming from OpenAI.  If I log into my OpenAI console, I see that there are no requests that use the API Key I supplied.

It appears the API key isn't being used.  Has anyone else had this issue?  I've tried both V1.0.13 and V1.0.12.    

I should add that I  have a paid OpenAI plan.

**Comment by RamiAwar** [comment, marked as answer]
Hey @sigott ! This is indeed the best place, if it's a bug I'll convert it to an issue 🙂 

That is strange because the fact that you entered your OpenAI key means that we already did calls to OpenAI to make sure your key is valid.

It might be that your quota is indeed zero and thus no API requests are being logged cause no requests are being made! Could you look into that? Make sure that you do indeed have quota, maybe use the key in another openai client to test it.

**Reply by sigott** [reply]
Thanks @RamiAwar 
It turns out I had an issue with my OpenAI account.  I had billing set up correctly but it wasn't working then suddenly just started to.  

Great project, BTW.  If I can free up some time, I would like to help with it.

**Reply by RamiAwar** [reply]
Glad to hear! Oh that would be amazing 😊
